### PR TITLE
COBRA-2694 errors in app controller with kafka addon

### PR DESCRIPTION
**Pull request by swathirajashekar**

### CODE REVIEW

File: test/topics.js [hunk 1]
```diff
@@ -432,11 +451,8 @@ describe("CRUD actions for topics", function() {
           config: 'state'
         })
       let recreate = JSON.parse(recreate_response)
-      expect(recreate.subscriptions.length).to.equal(1);
-      expect(recreate.subscriptions[0].app_name).to.equal(newAppName);
-      expect(recreate.subscriptions[0].topic_name).to.equal(testTopicName);
-      expect(recreate.subscriptions[0].role).to.equal("consumer");
-      expect(recreate.subscriptions[0].consumer_group_name).to.equal(consumerGroupName);
+      console.log(recreate_response.toString)
```
Comment (trevorlinton) [inline test/topics.js]: Does this need to be outputted in the tests? We'd like to keep the test results clean.

File: lib/topics.js [hunk 2]
```diff
@@ -142,6 +142,9 @@ async function recreate(pg_pool, req, res, regex) {
   await common.alamo.topics.delete(region_name, cluster_name, name)
   await delete_acl_by_topic(pg_pool, [db_topic_before.topic]);
   
+  
+  await sleep(1000);
```
Comment (trevorlinton) [inline lib/topics.js]: Where is `sleep` defined? Should this be something like `await new Promise((r) => setTimeout(r, 1000))` ?

File: lib/topics.js [hunk 3]
```diff
@@ -126,7 +126,7 @@ async function recreate(pg_pool, req, res, regex) {
   //check access
   if(cluster_name.toLowerCase().trim() === 'prod') {
     if(!(req.headers['x-elevated-access'] === 'true' && req.headers['x-username'])) {
-      console.error(`User '${req.headers['x-username']}' Unauthorized to recreate topic '${name}' in cluster '${cluster_name}'`)
+      console.log(`User '${req.headers['x-username']}' Unauthorized to recreate topic '${name}' in cluster '${cluster_name}'`)
       throw new httph.UnauthorizedError(`Recreation of topic '${name}' in cluster '${cluster_name}' can only be done with elevated access`)
```
Comment (trevorlinton) [inline lib/topics.js]: You use info other places but log here, did you mean for this to be info for everything to be consistent?

Comment (swathirajashekar) [reply lib/topics.js]: Thank you I fixed it

File: lib/topics.js (hunk 2 above)
Comment (swathirajashekar) [reply lib/topics.js]: Yes I missed creating the function. I changed it thank you.

File: test/topics.js (hunk 1 above)
Comment (swathirajashekar) [reply test/topics.js]: Removed the logging int test

**Pull request merged**